### ISSUE TITLE
No way to install silently?

### ISSUE THREAD

**Issue by traceurvostok** [Metabolix/HackBGRT#26]
Have not found how to install this silently with the default config using a command line switch.

Is this possible?

**Comment by NicolaPietro** [comment]
I have the same need.
Is this feature expected in future release?

Thank's.

**Comment by FREEWING-JP** [comment]
https://github.com/Metabolix/HackBGRT/issues/49#issuecomment-727183574

Is this Helps to you ?

**Comment by ottelo9** [comment]
My solution works for me (on a x64 Windows 10, UEFI):

First edit the config.txt line to:
`boot=\EFI\Microsoft\Boot\bootmgfw.efi`

Then the script (put all into the same folder):
```
@echo off

rem Drive name that does not currently exist in the system .
rem Generally "Y:" drives usually does not exist .
set EFI_DRIVE_LETTER=Y
set HACK_BGRT_PATH=%EFI_DRIVE_LETTER%:\EFI\HackBGRT

mountvol %EFI_DRIVE_LETTER%: /S

mkdir %HACK_BGRT_PATH%

rem You have to need Edit config.txt , Before copying it .
rem ex. boot=\EFI\Microsoft\Boot\bootmgfw.efi
copy %~dp0config.txt %HACK_BGRT_PATH%

rem You change to your favorite image file .
copy %~dp0splash.bmp %HACK_BGRT_PATH%

rem for 64bit System HackBGRT bootx64.efi
copy %~dp0bootx64.efi %HACK_BGRT_PATH%

dir %EFI_DRIVE_LETTER%:

mountvol %EFI_DRIVE_LETTER%: /D

rem Change boot loader to \EFI\HackBGRT\bootx64.efi
bcdedit /set {bootmgr} path \EFI\HackBGRT\bootx64.efi

rem Enjoy HackBGRT !!
pause
```

**Comment by Metabolix** [comment]
Implemented in version 2.0.0.
